src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/results.html.twig

```diff
@@ -22,6 +23,13 @@
                         <td>{{ elements.method }}</td>
                         <td>{{ elements.url }}</td>
                         <td>{{ elements.time|date('r') }}</td>
+                        <td>
+                        {% if elements.status_code is defined and elements.status_code %}
+                            <abbr title="{{ http_status_text(elements.status_code, 'unknown') }}">{{ elements.status_code }}</abbr>
```

Comment:
is it worth displaying the default status text here ? The displayed text is wrong anyway in case the profiled Response is using a custom status text, because you always display the default one